src/constants/DefaultValues.js

```diff
@@ -1,3 +1,15 @@
 export const defaultTwitBot = {
   post: (action, data, cb) => cb(),
 };
+
+export const recordAudioMessage = (word) => {
+  const messages = [
+    `Can you pronounce the word "${word}"? Reply with an audio recording! 🎤 \n#Igbo #LearnIgbo`,
+    `Let's hear you pronounce this word "${word}", \
```

Comment:
Are these `\` necessary for new lines? Do they show up in the final tweet? I thought template literals automatically handle new lines

Reply:
They're not necessary and do not show up in the final tweet. But I'll take them out and just back indent as necessary so the text is fine when it's sent out.